test/rule-matches/not-html-matches.js

```diff
@@ -5,20 +5,21 @@ describe('not-html-matches', function() {
   var rule;
 
   beforeEach(function() {
-    rule = axe._audit.rules.find(function(rule) {
-      return rule.id === 'valid-lang';
-    });
+    axe.teardown();
+    rule = axe.utils.getRule('valid-lang');
   });
 
   afterEach(function() {
     fixture.innerHTML = '';
   });
 
   it('returns true when element is not the html element', function() {
-    assert.isTrue(rule.matches(fixture));
+    var vNode = axe.setup(fixture);
+    assert.isTrue(rule.matches(null, vNode));
   });
 
   it('returns false when element is the html element', function() {
-    assert.isFalse(rule.matches(document.documentElement));
+    var vNode = axe.setup(document.documentElement);
```

Comment:
I wonder if we should make document.documentElement the default argument for setup. WDYT?

Reply:
[It already is](https://github.com/dequelabs/axe-core/blob/develop/lib/core/utils/get-flattened-tree.js#L142) since `axe.utils.getFlattenTree` defaults to `document.documentElement`